src/lang/en-US.json

```diff
@@ -433,8 +433,10 @@
   "subheading_team_elo_rankings": "k=32, init=1000",
   "hero_pro_tab": "Professional",
   "hero_public_tab": "Public",
+  "hero_turbo_tab": "Turbo",
   "hero_pro_heading": "Heroes in Professional Matches",
   "hero_public_heading": "Heroes in Public Matches (Sampled)",
+  "hero_turbo_heading": "Heroes in Turbo Matches (Sampled)",
```

Comment:
this isn't sampled, it's all turbo games

Reply:
I made a comment to myself then forgot to poke you about it. will fix it.